pkg/dhcp6client/dhcp6client.go

```diff
@@ -92,7 +96,7 @@ func (p *buffer) remaining() []byte {
 
 func (c *Client) ReadReply() (*dhcp6.Packet, error) {
 	var err error
-	for i := 0; i < 5; i++ { // five attempts
+	for i := 0; i < c.attempts; i++ { // five attempts
 		pb := make([]byte, 1500)
```

Comment:
Chris is right. 576 is the minimum legal value of max DHCPv6 size option.
See https://tools.ietf.org/html/draft-ietf-dhc-v6opts-00#page-11